server_config_parser.h

```diff
@@ -2,11 +2,37 @@
 #define SERVER_CONFIG_PARSER_H
 
 #include "config_parser.h"
+#include "http_handler_echo.h"
+#include "http_handler_file.h"
 
+struct server_config {
+    int port;
```

Comment:
if you change the other instance of port for the server into a short, change this into a short as well